pkg/gadgets/helpers_all.go

```diff
@@ -33,3 +33,26 @@ func ProtoString(proto int) string {
 	}
 	return protoStr
 }
+
+func FromCString(in []byte) string {
```

Comment:
Commit 00b7ff3873ce57898a6e31ba5122ba0647cdea07 can be a separate PR.

Reply:
Moved to #2556